swagger.yaml

```diff
@@ -1907,60 +1907,80 @@ definitions:
       href:
         description: "A URL to the version this resource relates to"
         type: string
-  ObservationEndpoint:
-    description: "An object containing information on a single observation for a given version of a dataset"
+  ObservationsEndpoint:
+    description: "An object containing information on a list of observations for a given version of a dataset"
     type: object
     properties:
-      context:
-        description: ""
-        type: string
       dimensions:
         description: "A list of dimensions for the given query"
         type: object
         properties:
-          <dimension>:
+          <dimension name>:
             type: object
             properties:
-              options:
+              option:
                 description: "A list of links to the corresponding dimension codes for the given `dimension_options`"
-                type: array
-                items:
+                type: object
+                properties:
+                  href:
+                    description: "A link to the corresponding dimension code for the given `dimension_option` (query parameter - e.g. 'geography: K02000001'"
```

Comment:
Not sure the e.g. is representative, if we're trying to give an example of this field we should use the swagger example field with reasonable values?

Reply:
So the e.g. bit is an example of the query parameter in brackets which dictates whether the dimension and hence dimension_option exist in this part of the document. I probably agree with you that this is probably not the best place to put an example of query parameters.